src/System.Net.Requests/tests/System.Net.Requests.Tests.builds

```diff
@@ -2,14 +2,15 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
+    <Project Include="System.Net.Requests.Tests.csproj"/>
     <Project Include="System.Net.Requests.Tests.csproj">
-      <OSGroup>Unix</OSGroup>
```

Comment:
Is it safe to assume we don't no longer have a Unix specific build of this test project, as you appear to have removed that configuration.